NaZdrowie/src/screens/PatientScreens/QrScannerScreen.tsx

```diff
@@ -11,7 +11,7 @@ import {
   StatusBar,
   StyleSheet,
 } from "react-native";
-import { useBindPatientToDoctor } from "services/patientData";
+import { useBindPatientToDoctor } from "services/universalData";
```

Comment:
I feel like this name is a bit too general. Let's think of smth more precise, or move it to patientData - which also is "universal" in many aspects. We can think of dividing patient and doctor data, but let's not use no meaningful names. We may separate patient/doctor datas by logic, like some different class for referrals/results, etc

Reply:
But as for now you can just put it inside doctorData/patientData - we will think of it